clang/test/CIR/CodeGen/builtin-floating-point.c

```diff
@@ -14,15 +14,15 @@ double my_ceil(double f) {
   // CHECK: {{.+}} = cir.ceil {{.+}} : !cir.double
 }
 
-// long double my_ceill(long double f) {
-//   return __builtin_ceill(f);
-//   // DISABLED-CHECK: cir.func @my_ceill
-//   // DISABLED-CHECK: {{.+}} = cir.ceil {{.+}} : f80
-// }
+long double my_ceill(long double f) {
+  return __builtin_ceill(f);
+  // CHECK: cir.func @my_ceill
+  // CHECK: {{.+}} = cir.ceil {{.+}} : !cir.long_double<!cir.f80>
```

Comment:
Similar here, there are no CIRGen tests for PPC and FP128Type, please add those.

Reply:
I have removed `!cir.f128` and `!cir.ppc_doubledouble` from this PR because their targets are not yet supported by CIR. See my latest comment.